ggcore/client.py

```diff
@@ -83,39 +89,37 @@ def __init__(self, bootstrap_config):
         self._token_factory = TokenFactory(
             self._security_client.request_and_store_token)
 
-    def make_request(self,
-                     sdk_request: SdkServiceRequest) -> SdkServiceResponse:
-        """Define make_request that adds bearer auth headers for the sdk
-        request.
-        """
+    def build_sdk_request(self,
+                          api_def: AbstractApi) -> SdkServiceRequest:
```

Comment:
Looks like this is missing a doc-string. I realize now that it appears [this line](https://github.com/graphgrid/graphgrid-sdk-python/blob/2.0/scripts/ci/install-and-run-tests#L30) results in only fatal and error messages being reported whereas PEP 8 related errors seem to be reported as convention error messages. So if you remove the `% 4` the linter should pick these up going forward. This [doc page](https://docs.pylint.org/en/1.6.0/run.html#exit-codes) lists out the error codes. However, I'm not sure if it makes the most sense to simply remove it, or change it to something like `% 32`, thoughts?

Reply:
After doing some local testing I think this isn't getting a missing doc-string linting error because the superclass method has a doc-string on it already.  I think this is expected behavior?